public/js/main-template.js

```diff
@@ -12,93 +12,122 @@ $(document).ready(function () {
     asideBarRight = document.getElementById("AsideBarRight");
     fieldContainer = document.getElementById("field-container");
     installcontainer = document.getElementById("install-container");
+   if (document.getElementById("pannelone")) {
+        document.getElementById("pannelone").classList.toggle("Active");
+        document.getElementById("AsideBarRight").classList.toggle("RightPanel1Active");

```

Comment:
these two also just read on top before if 

```suggestion

const AsideBarRight=document.getElementById("AsideBarRight");
        AsideBarRight.classList.toggle("RightPanel1Active");

```